### PR TITLE
feat: add aggregating mvs for dependency graph

### PR DESCRIPTION
**Pull request by srikanthccv**
Required for https://github.com/SigNoz/signoz/pull/1467

### CODE REVIEW

File: exporter/clickhousetracesexporter/migrations/000016_dependency_graph_minutes.up.sql [hunk 1]
```diff
@@ -0,0 +1,49 @@
+CREATE TABLE IF NOT EXISTS signoz_traces.dependency_graph_minutes (
+    src LowCardinality(String) CODEC(ZSTD(1)),
+    dest LowCardinality(String) CODEC(ZSTD(1)),
+    duration_quantiles_state AggregateFunction(quantiles(0.5, 0.75, 0.9, 0.95, 0.99), Float64) CODEC(Default),
+    error_count SimpleAggregateFunction(sum, UInt64) CODEC(T64, ZSTD(1)),
+    total_count SimpleAggregateFunction(sum, UInt64) CODEC(T64, ZSTD(1)),
+    timestamp DateTime CODEC(Delta(4), Default)
```
Comment (makeavish) [inline exporter/clickhousetracesexporter/migrations/000016_dependency_graph_minutes.up.sql]: Any specific reason why we are using Delta(4)? 
We are using CODEC(DoubleDelta, LZ4) for logs and traces as [this](https://altinity.com/blog/2019/7/new-encodings-to-improve-clickhouse) benchmark by altinity shows that DoubleDelta is more efficient.

**Comment by makeavish** [comment]
@ankitnayan We should do a benchmark of how adding new materialized views is affecting insert performance as we have added 5-6 new views recently.

**Review by makeavish — DISMISSED** [review]

**Comment by ankitnayan** [comment]
> 

an issue for this would be needed

File: exporter/clickhousetracesexporter/migrations/000016_dependency_graph_minutes.up.sql (hunk 1 above)
Comment (srikanthccv) [reply exporter/clickhousetracesexporter/migrations/000016_dependency_graph_minutes.up.sql]: Updated

**Review by makeavish — APPROVED** [review]

**Review by ankitnayan — APPROVED** [review]

**Pull request merged**